### PR TITLE
merge 0.3.0 code to develop

### PR DESCRIPTION
**Pull request by grawk**
new develop branch, adding 0.3.0 code into main repo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,18 +20,13 @@
...
     "nemo-drivex": "^0.1.0",
     "nemo-locatex": "^0.1.0",
     "nconf": "*",
     "nemo-shared-objects": "^0.2.0"
   },
-  "peerDependencies": {
-    "nemo-drivex": "^0.1.0",
-    "nemo-locatex": "^0.1.0",
-    "nemo": "^0.2.0"
-  },
```
Comment (nikulkarni) [inline package.json]: I thought this is still required, albeit updated version semantics?

Comment (grawk) [inline package.json]: Yes I want to address that before merging. Thanks for making a note of it.

**Comment by grawk** [comment]
@nikulkarni I added back in the peerDependency section. It was removed during pre-release development since semver forces you to enter the exact pre-release version and it was slowing integration testing

**Comment by nikulkarni** [comment]
:+1:

**Pull request merged**